CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 #### Fixes
 
+* [#455](https://github.com/ruby-grape/grape-swagger/pull/455): Passing `Array[<primitive>]` should create json indicating `array` type - [@tyspring](https://github.com/tyspring).
```

Comment:
Since this is a changelog, instead of "should create" make it "generates `array` type for Array[Class]" for example.